src/Process/3dTilesProcessing.js

```diff
@@ -257,24 +257,36 @@ export function pre3dTilesUpdate(context, layer) {
 
 const boundingVolumeBox = new THREE.Box3();
 const boundingVolumeSphere = new THREE.Sphere();
-export function computeNodeSSE(camera, node) {
+export function computeNodeSSE(context, camera, node) {
```

Comment:
context used only for distance, so the params could be with the `distance`.

Reply:
Yeah.. or I could remove camera (since it's part of context as well).

Reply:
it's preferable to pass only the parameters used in the function